librosa/filters.py

```diff
@@ -825,6 +815,40 @@ def wavelet_lengths(
     return lengths, f_cutoff
 
 
+def _relative_bandwidth(*, freqs: np.ndarray) -> np.ndarray:
```

Comment:
plural, maybe? `_relative_bandwidths`?

Reply:
We could, though I don't think it matters much as a private function.  The meaning seems clear enough either way, assuming we're okay with the "relative bandwidth" terminology?

Reply:
it's just that the input (`freqs`) is plural and we can't make the output name plural (`alpha`) since it's sometime singular. so pluralizing the method name would be helpful to understand that this is a many-to-many operation